.github/workflows/tox.yml

```diff
@@ -12,15 +12,23 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
     strategy:
       matrix:
         python-version: ["3.6", "3.8"]
+        os: [ubuntu-latest, ubuntu-22.04, ubuntu-20.04]
+        exclude: 
+          - python-version: "3.10"
+            os: ubuntu-20.04
+          - python-version: "3.6"
+            os: ubuntu-22.04
+          - python-version: "3.6"
+            os: ubuntu-latest
+    runs-on: ${{matrix.os}}
```

Comment:
<img width="841" alt="image" src="https://github.com/StackStorm/orquesta/assets/1533818/d0d0904e-b369-497e-850b-1332c964ca5c">


Because we're testing python and not OS at this level, from this build matrix, I think it makes sense to just have 1 environment for reach python version:
- 3.6, ubuntu-20.04
- 3.8, ubuntu-22.04

should simplify it a bit, unless you had specific reason to have a wider build matrix

Reply:
Here https://github.com/StackStorm/orquesta/pull/256/files#diff-8a1944567ec5d9e12aaaac28489c47e6567e13b58d861815379f9b8d3aeb2623R15 the build was green by using ubuntu-20.04 environment for both py3.6 and py3.8.
Maybe that's the reason of so many unit test fails in this PR build? Worth checking.

Reply:
@AndroxxTraxxon  The CI is fixed here https://github.com/StackStorm/orquesta/pull/257 and is now ✅  on master, thanks to @amanda11 and @nzlosh.

Please sync your work with the master branch.